src/main/java/core/basesyntax/service/impl/FruitShopDataProcessorService.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.handler.ShopOperationHandler;
+import core.basesyntax.service.DataProcessorService;
+import core.basesyntax.strategy.ShopOperationStrategy;
+import java.util.List;
+
+public class FruitShopDataProcessorService implements DataProcessorService {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final String COMMA = ",";
```

Comment:
```suggestion
    private static final String SEPARATOR = ",";
```
when the format will change from , to : for example, you will have to change the constant value + constant name, since : isn't a comma :)

Reply:
Done